### ISSUE TITLE
New `log evidence` field breaks `Chain` type in TuringLang/Turing

### ISSUE THREAD

**Issue by trappmartin** [TuringLang/MCMCChains.jl#21]

**Comment by cpfiffer** [comment]
What was the resolution to this issue? I just got the bug elsewhere. Looks like it came from a `describe` call.

```
ERROR: LoadError: type Chain has no field logevidence
Stacktrace:
 [1] getproperty(::Any, ::Symbol) at ./sysimg.jl:18
 [2] header(::Turing.Chain{AbstractRange{Int64}}) at /home/cameron/.julia/dev/MCMCChain/src/chains.jl:274
 [3] #summarystats#74(::Symbol, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Turing.Chain{AbstractRange{Int64}}) at /home/cameron/.julia/dev/MCMCChain/src/stats.jl:119
 [4] (::getfield(StatsBase, Symbol("#kw##summarystats")))(::NamedTuple{(:etype,),Tuple{Symbol}}, ::typeof(StatsBase.summarystats), ::Turing.Chain{AbstractRange{Int64}}) at ./none:0
 [5] #describe#64(::Array{Float64,1}, ::Symbol, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Base.TTY, ::Turing.Chain{AbstractRange{Int64}}) at /home/cameron/.julia/dev/MCMCChain/src/stats.jl:64
 [6] describe(::Base.TTY, ::Turing.Chain{AbstractRange{Int64}}) at /home/cameron/.julia/dev/MCMCChain/src/stats.jl:64
 [7] #describe#63(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Turing.Chain{AbstractRange{Int64}}) at /home/cameron/.julia/dev/MCMCChain/src/stats.jl:56
 [8] describe(::Turing.Chain{AbstractRange{Int64}}) at /home/cameron/.julia/dev/MCMCChain/src/stats.jl:56
 [9] top-level scope at none:0
 [10] include_string(::Module, ::String, ::String) at ./loading.jl:1002
 [11] (::getfield(Atom, Symbol("##128#133")){String,String,Module})() at /home/cameron/.julia/packages/Atom/Fha1N/src/eval.jl:120
 [12] withpath(::getfield(Atom, Symbol("##128#133")){String,String,Module}, ::String) at /home/cameron/.julia/packages/CodeTools/hB4Hy/src/utils.jl:30
 [13] withpath at /home/cameron/.julia/packages/Atom/Fha1N/src/eval.jl:46 [inlined]
 [14] #127 at /home/cameron/.julia/packages/Atom/Fha1N/src/eval.jl:117 [inlined]
 [15] hideprompt(::getfield(Atom, Symbol("##127#132")){String,String,Module}) at /home/cameron/.julia/packages/Atom/Fha1N/src/repl.jl:84
 [16] macro expansion at /home/cameron/.julia/packages/Atom/Fha1N/src/eval.jl:116 [inlined]
 [17] (::getfield(Atom, Symbol("##126#131")){Dict{String,Any}})() at ./task.jl:85
in expression starting at /home/cameron/code/TuringTutorials/regression.jl:67
```

**Comment by trappmartin** [comment]
Good catch. I only resolved it for the show method.

**Comment by trappmartin** [comment]
I'll comment out the problematic line for now. Once we have removed the Chain type in Turing, this line should be uncommented again.

Leaving this issue open for now.

**Comment by cpfiffer** [comment]
Fixed by #45.
